src/elements/content-sidebar/versions/VersionsItem.scss

```diff
@@ -1,4 +1,5 @@
 @import '../../common/variables';
+@import '../../../styles/constants/buttons';
```

Comment:
nit:
an explicit import isn't needed since the main `variables` file will import this:
https://github.com/box/box-ui-elements/blob/master/src/styles/_variables.scss#L7